ui/app/models/database/role.js

```diff
@@ -145,6 +167,12 @@ export default class RoleModel extends Model {
       'rollback_statements',
       'renew_statements',
     ];
+
+    // remove enterprise-only attrs if on community
```

Comment:
I see we do this in a couple of the models, so no need to change. But for future iterations, we're moving away from adding logic to the models. The goal is to keep them as close to a one-to-one schema of the api as possible. And you'll see how this will become even more so as we remove models completely with Jordan's work. 

All that to say, if logic can go in the component instead of the model we should move it there. And in this case, we could use the this.version.isEnterprise check in the component hbs file rather than here in the model. A heads up, not a request to change.